packages/vue/cypress/integration/happy-path.js

```diff
@@ -0,0 +1,369 @@
+    cy.iframe().find(selectors.hero.nextArrowButton).click({ multiple: true });
+  });
+  it("test SfCallToAction component", () => {
+    cy.iframe().find(selectors.callToAction.button).click();
+    cy.iframe().find(selectors.hero.nextArrowButton).click({ multiple: true });
+  });
+  it("test SfCarousel component", () => {
+    cy.iframe().find(".sf-carousel-item:not(.glide__slide--clone)").first().should("have.class", "glide__slide--active").find(selectors.carousel.productCard);
+    cy.iframe().find(selectors.carousel.prevArrowButton).click({ multiple: true });
+    cy.iframe().find(".sf-carousel-item:not(.glide__slide--clone)").first().should("not.be.visible");
+    cy.iframe().find(selectors.carousel.nextArrowButton).click({ multiple: true });
+    cy.iframe().find(".sf-carousel-item:not(.glide__slide--clone)").first().find(selectors.carousel.productCard).should("be.visible");
+    cy.iframe().find(selectors.carousel.productCard).first().find(selectors.carousel.productCardWishlistButton).should("not.have.class", "on-wishlist").click({force:true})
+    cy.iframe().find(selectors.carousel.productCard).first().find(selectors.carousel.productCardAddIcon).click({ force: true })
+  });
+  it("test Instagram Section component", () => {
+    cy.iframe().find(selectors.instagramSection.images).find(".sf-image--overlay");
+  });
+  it("test SfBanner component", () => {
+    cy.iframe().find(selectors.banner.banner).find(selectors.banner.buttons).first().click().find(".sf-image");
+    cy.iframe().find(selectors.banner.banner).last(selectors.banner.buttons).click().find(".sf-image");
+  });
+});
+
+describe("Checkout page", () => {
+  beforeEach(() => {
+    cy.visit("/?path=/docs/pages-checkout--common");
+  });
+  it("test Details step", () => {
+    cy.iframe().find(selectors.details.loginButton).click();
+    cy.iframe().find(selectors.details.firstNameInput).type("John").should("have.class", "has-text");
+    cy.iframe().find(selectors.details.lastNameInput).type("Doe").should("have.class", "has-text");
+    cy.iframe().find(selectors.details.emailInput).type("john.doe@vsf.io").should("have.class", "has-text");
+    cy.iframe().find(`${selectors.details.createAccountCheckbox} input`).check({force: true})
+    cy.iframe().find(selectors.details.createAccountCheckbox).should("have.class", "is-active")
+    cy.iframe().find(selectors.details.createPasswordInput).should("be.visible") 
+      .parents(".sf-input").type("password").should("have.class", "has-text")
+      .find(".sf-icon").click().should("not.have.class", "hidden");
+    cy.iframe().find(selectors.summary.promoCode).type("1a2b1c2d1e2f").should("have.class", "has-text");
+    cy.iframe().find(selectors.summary.applyButton).click();
+    cy.iframe().find(selectors.checkout.nextButton).contains("Go to shipping").click();
+    // cy.iframe().find(selectors.checkout.stepsProgress).should("have.css", "--_steps-progress-active-step", "2");
```

Comment:
why is it as a comment?

Reply:
I couldn't find solution to properly check for style attribute, but finally I did and it works.